packages/core/ios/RNSentryReplay.m

```diff
@@ -31,14 +32,25 @@ + (void)updateOptions:(NSMutableDictionary *)options
             @"redactAllImages" : replayOptions[@"maskAllImages"] ?: [NSNull null],
             @"redactAllText" : replayOptions[@"maskAllText"] ?: [NSNull null],
             @"maskedViewClasses" : [RNSentryReplay getReplayRNRedactClasses:replayOptions],
+            @"unmaskedViewClasses" : [RNSentryReplay getReplayRNUnmaskClasses],
         }
     }
                forKey:@"experimental"];
 }
 
++ (NSArray *_Nonnull)getReplayRNUnmaskClasses
+{
+    // We can't import RNSentryReplayUnmask.h here because it's Objective-C++
+    // To avoid typos, we test the class existens in the tests
+    return @[ @"RNSentryReplayUnmask" ];
+}
+
 + (NSArray *_Nonnull)getReplayRNRedactClasses:(NSDictionary *_Nullable)replayOptions
 {
     NSMutableArray *_Nonnull classesToRedact = [[NSMutableArray alloc] init];
+    // We can't import RNSentryReplayMask.h here because it's Objective-C++
+    // To avoid typos, we test the class existens in the tests
+    [classesToRedact addObject:@"RNSentryReplayMask"];
```

Comment:
For Unmask to work this PR requires changes from https://github.com/getsentry/sentry-cocoa/pull/4522

Add add call of `PrivateSentrySDKOnly.setIgnoreWrapperClass(IgnoreWrapper.self)`